app/assets/javascripts/govuk_publishing_components/analytics/explicit-cross-domain-links.js

```diff
@@ -85,5 +68,22 @@
     }
   }
 
-  global.GOVUK = GOVUK
-})(window)
+  ExplicitCrossDomainLinks.prototype.init = function () {
+    this.element = this.$module
```

Comment:
Seems like an unnecessary reassignment of a variable - could the script be updated to use `this.$module` instead of `this.element`?

Reply:
Good spot, no idea why I did this, will update!